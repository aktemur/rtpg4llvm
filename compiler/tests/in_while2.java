/*stmt*/
while(x < 10) {
    if(true)
	continue;
    x++;
    break;
}
