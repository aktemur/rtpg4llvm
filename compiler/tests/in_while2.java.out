new WhileStmt(mkBinOp(Less,
         new VarExpr("x"),
         mkConst(10)),
      mkScopedStmt(3, new IfElseStmt(mkConst(true),
      new ContinueStmt()),
      new ExprStmt(mkUnaryPostOp(PlusPlus, new VarExpr("x"))),
      new BreakStmt()))
